recipes/libelf/all/test_package/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
 
 add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::libelf)
```

Comment:
this package has no official cmake modules/config, so we did not define any in package_info. As far as I understand it means conan 2.0 won't generate any cmake module or config file, event with the generators enabled right ?
So in the end this package won't be consumable from cmake in conan 2.0 ?

Reply:
>  So in the end this package won't be consumable from cmake in conan 2.0 ?

yes.

you still can generate based on package name, it will be `libelf::libelf`